plugins/out_http/http.c

```diff
@@ -112,7 +112,10 @@ static int http_post(struct flb_out_http *ctx,
                         payload_buf, payload_size,
                         ctx->host, ctx->port,
                         ctx->proxy, 0);
-
+    if (!c) {
+        flb_plg_error(ctx->ins, "[http_client] failed to create HTTP client");
```

Comment:
thanks.

Note that returning at this point might leak memory from the allocations above, so the PR will need to take care of that too.

Reply:
fixed